promdens.py

```diff
@@ -189,7 +192,7 @@ def calc_field_envelope(self, t):
                     field[k] = np.sin(np.pi/2*(t[k] - self.field_t0 + self.field_fwhm)/self.field_fwhm)
             return field
         elif self.field_envelope_type == 'sin2':
-            T = 1.373412575*self.field_fwhm
+            T = 1/(2 - 4/np.pi*np.arcsin(2**(-1/4)))*self.field_fwhm
```

Comment:
Nice, this is a great change. One the paper is on the arxiv, we should perhaps include links to the eqations in the paper.